ompi/mca/coll/tuned/coll_tuned_reduce_decision.c

```diff
@@ -42,6 +43,8 @@ static const mca_base_var_enum_value_t reduce_algorithms[] = {
     {5, "binomial"},
     {6, "in-order_binary"},
     {7, "rabenseifner"},
+    {8, "knomial"},
```

Comment:
twice ?

Reply:
This is the knomial implementation for reduce.  The other one is Bruck's method for allgather with any radix k.